init-tools.sh

```diff
@@ -64,6 +64,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

Comment:
We almost do. The porting is currently in progress and only about 17 CoreCLR tests out of the about 8000 are failing.